setup.py

```diff
@@ -45,6 +45,7 @@
     },
     "develop": {
         "pytest",
+        "pytest-benchmark"
```

Comment:
Probably not needed yet, but if the list of dependencies only used in automated systems grows, you could add a new target to capture those and separate them from the ones we'll all install for development

Reply:
that makes sense